sass/_themes.scss

```diff
@@ -32,6 +32,7 @@ $themes: (
     controlGrid:            #fafafb,
     controlDialogHead:      #FFFFFF,
     controlDialogBody:      #FAFAFA,
+    controlDialogButton:    #2a3052,
```

Comment:
Another colour just for this button? We need to get our designers to stick to a certain palette :/ 

Can you use `activeColor` and see if Afiq is ok with that? It's same colour as the modal dialog buttons.